remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/ProtobufMetadataManagerImpl.java

```diff
@@ -116,6 +124,7 @@ private void processProtostreamSerializationContextInitializers(ClassLoader clas
    private void processSerializationContextInitializer(Iterable<SerializationContextInitializer> initializers) {
       if (initializers != null) {
          for (SerializationContextInitializer sci : initializers) {
+            log.debugf("Registering SerializationContextInitializer %s", sci.getClass().getName());
             try {
                sci.registerSchema(serCtx);
                sci.registerMarshallers(serCtx);
```

Comment:
I have found a bigger problem (not introduced by this PR). We're currently not registering the loaded SCIs with the `SerializationContextRegistry` [ISPN-12093](https://issues.redhat.com/browse/ISPN-12093)